src/extractor.py

```
@@ -0,0 +1,22 @@
+        caminho_pdf: O caminho para o arquivo PDF.
+
+    Returns:
+        O conteúdo de texto do PDF como uma string.
+    """
+    try:
+        doc = fitz.open(caminho_pdf)
+        texto = ""
+        for pagina in doc:
+            texto += pagina.get_text()
+        doc.close()
+        return texto
```

Ruff (0.15.1)

[warning] Consider moving this statement to an `else` block

(TRY300)